Integrations/EclecticIQ/EclecticIQ.py

```diff
@@ -0,0 +1,640 @@
+    res = requests.request(
+        method,
+        SERVER + url_suffix,
+        headers=headers,
+        json=cmd_json,
+        proxies=PROXIES,
+        verify=USE_SSL
+    )
+    if res.status_code not in {200}:
+        if res.status_code == 405:
+            return_error(
+                'Error in API call to EclecticIQ Integration: [405] - Not Allowed - Might occur cause of an invalid '
+                'URL.')
+        return_error(res.content)
+        errors = json.loads(res.text).get('errors', {})[0]
+        title = errors.get('title', '')
+        detail = errors.get('detail', '')
+        return_error('Error in API call to EclecticIQ Integration: [%d] - %s - %s' % (res.status_code, title, detail))
+    try:  # Verify we can get json from the response
+        return res.json()
+    except ValueError:
+        return_error(res)
+
+
+def maliciousness_to_dbotscore(maliciousness, threshold):
+    """
+
+    Translates EclecticIQ obversable maliciousness confidence level to DBotScore based on given threshold
+
+    Parameters
+    ----------
+    maliciousness : str
+        EclecticIQ obversable maliciousness confidence level.
+    threshold : str
+        Minimum maliciousness confidence level to consider the IOC malicious.
+
+    Returns
+    -------
+    number
+        Translated DBot Score
+
+    """
+    maliciousness_list = ['unknown', 'safe', 'low', 'medium', 'high']
+
+    maliciousness_dictionary = {
+        'unknown': 0,
+        'safe': 1,
+        'low': 2,
+        'medium': 2,
+        'high': 3
+    }
+
+    for i in maliciousness_list[maliciousness_list.index(threshold):]:
+        maliciousness_dictionary[i] = 3
+
+    return maliciousness_dictionary[maliciousness]
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def test_module():
+    """
+
+    The function which runs when clicking on Test in integration settings
+
+
+    Returns
+    -------
+    str
+        ok if getting observable successfully
+
+    """
+
+    get_observable('8.8.8.8')
+    demisto.results('ok')
+
+
+def login():
+    """
+
+    Logins to EclecticIQ API with given credentials and sets the returned token in the headers
+
+    """
+
+    cmd_url = '/api/auth'
+    cmd_json = {
+        'password': PASSWORD,
+        'username': USERNAME
+    }
+    response = http_request('POST', cmd_url, cmd_json=cmd_json)
+    if 'token' in response:
+        token = response['token']
+    else:
+        return_error('Failed to retrieve token')
+    HEADERS['Authorization'] = 'Bearer {}'.format(token)
+
+
+def ip_command():
+    """
+
+    Gets reputation of an EclecticIQ IPv4 observable
+
+    Parameters
+    ----------
+    ip : str
+        IPv4 to get reputation of
+
+    Returns
+    -------
+    entry
+        Translated DBot Score
+
+    """
+
+    ip = demisto.args()['ip']
+
+    response = get_observable(ip)
+
+    if 'total_count' in response and response['total_count'] == 0:
+        human_readable = 'No results found'
```

Comment:
why not leave it the tableToMarkdown?

Reply:
i dont like the way tabletomarkdown handles it, because it shows a big title and then in small font size that there were no entries found